module.c

```diff
@@ -259,6 +262,42 @@ static PyObject *quickjs_to_python(ContextData *context_obj, JSValue value) {
 		JSValue exception = JS_GetException(context);
 		const char *cstring = JS_ToCString(context, exception);
 		const char *stack_cstring = NULL;
+		PyObject *py_stack = NULL;
+		if (context_obj->last_python_error_value == NULL || cstring == NULL || strstr(cstring, "Python call failed.") == NULL) {
```

Comment:
The check `strstr(cstring, "Python call failed.")` is quite brittle. We should not rely on the specific human-readable message of exceptions.

Reply:
I agree in principle, but how else to catch the error at https://github.com/PetterS/quickjs/blob/c603994a033b1414010e5896ec65495b9b9f2ef2/module.c#L629 specifically? Use another error type?